crosswalks/Citation File Format 1.2.0.csv

```diff
@@ -1,76 +1,68 @@
-Property,Citation File Format Core (CFF-Core) 1.0.2
-codeRepository,repository-code
-programmingLanguage,
-runtimePlatform,
-targetProduct,
-applicationCategory,
-applicationSubCategory,
-downloadUrl,repository-artifact
-fileSize,
-installUrl,
-memoryRequirements,
-operatingSystem,
-permissions,
-processorRequirements,
-releaseNotes,
-softwareHelp,
-softwareRequirements,
-softwareVersion,
-storageRequirements,
-supportingData,
-author,authors
-citation,
-contributor,
-copyrightHolder,
-copyrightYear,
-dateCreated,
-dateModified,
-datePublished,date-released
-editor,
-encoding,
-fileFormat,
-funder,
-keywords,keywords
-license,license/license-url
-producer,
-provider,
-publisher,
-sponsor,
-version,version
-isAccessibleForFree,
-isPartOf,references
-hasPart,
-position,
-description,abstract
-identifier,doi
-name,title
-sameAs,
-url,url
-relatedLink,
-givenName,person.given-names
-familyName,person.name-particle + person.family-names + person.name-suffix
-email,person.email/entity.email
-affiliation,person.affiliation
-identifier,person.orcid / entity.orcid
-name,entity.name
-address,person.address + person.city + person.region + person.post-code + person.country / entity.address + entity.city + entity.region + entity.post-code + entity.country
-type,reference.type
-id,
-softwareSuggestions,
-maintainer,
-contIntegration,
-buildInstructions,
-developmentStatus,
-embargoDate,
-funding,
-issueTracker,repository
-referencePublication,references
-readme,
-,
-,
-,
-,
-,
-,
-,
-,
+Property,Citation File Format (1.2.0)
+codeRepository,repository-code
+programmingLanguage,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,repository-artifact
+fileSize,
+installUrl,
+memoryRequirements,
+operatingSystem,
+permissions,
+processorRequirements,
+releaseNotes,
+softwareHelp,
+softwareRequirements,references
+softwareVersion,version
+storageRequirements,
+supportingData,
+author,authors
+citation,Metadata in the CFF root document / preferred-citation / references
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,
+dateModified,
+datePublished,date-released
+editor,
+encoding,
+fileFormat,
+funder,
+keywords,keywords
+license,license / license-url
+producer,
+provider,
+publisher,
+sponsor,
+version,version
+isAccessibleForFree,
+isPartOf,(identifiers)
+hasPart,(identifiers)
+position,
+description,abstract
+identifier,identifiers / person.orcid / doi
+name,title
+sameAs,identifiers.url
+url,url
+relatedLink,url
```

Comment:
as well as (some of?) `repository`, `repository-code`, `repository-artifact`, `identifeirs[i].type==url`

Reply:
Depends much on the interpretation of `relatedLink` semantics I guess. In any case, I've added it to the CSV and documentation in 08483bc83f2da6209490fd6d11541de643b42b98.